babel/messages/pofile.py

```diff
@@ -521,14 +522,21 @@ def _write(text):
             text = text.encode(catalog.charset, 'backslashreplace')
         fileobj.write(text)
 
+<<<<<<< HEAD
+    def _write_comment(comment, prefix='':
+=======
     def _write_comment(comment, prefix=''):
-        # xgettext always wraps comments even if --no-wrap is passed;
-        # provide the same behaviour
-        if width and width > 0:
-            _width = width
-        else:
-            _width = 76
-        for line in wraptext(comment, _width):
+>>>>>>> 67aa652a1c10bfdc92587cdae87fdab65d633b23
+        # NEVER wrap comments, this observation: "xgettext always wraps comments even if --no-wrap is passed;" is FALSE. There seemed to be a bug in the xgettext code, because wrapping doesn't always occur
+        # Make sure comments are unique and sorted alphabetically so locations can be easily searched and identify
+        has_comment = (bool(comment) and len(comment) > 0)
+        if not has_comment:
```

Comment:
I believe this simplifies to

```suggestion
        if not comment:
```

Reply:
Thank you for this comment. I'm changing the code and pushing another commit.